WalletWasabi.Fluent/Models/UI/QrGenerator.cs

```diff
@@ -0,0 +1,15 @@
+using System.Reactive.Concurrency;
+using System.Reactive.Linq;
+using Gma.QrCodeNet.Encoding;
+
+namespace WalletWasabi.Fluent.Models.UI;
+
+public class QrGenerator : IQrCodeGenerator
+{
+	private readonly QrEncoder _encoder = new();
+
+	public IObservable<bool[,]> Generate(string data)
+	{
+		return Observable.Start(() => _encoder.Encode(data).Matrix.InternalArray, DefaultScheduler.Instance);
+	}
```

Comment:
Why does this have to be an Observable and not just return the array?

Reply:
@soosr `Observable.Start(f,  DefaultScheduler.Instance);` is equivalent to running a new `Task` in the default scheduler (Thread Pool Task). The advantage is that we can bind this directly to a ViewModel property using `Bind`